python/tskit/util.py

```diff
@@ -592,3 +592,33 @@ def random_nucleotides(length: numbers.Number, *, seed: Union[int, None] = None)
     encoded_nucleotides = np.array(list(map(ord, "ACTG")), dtype=np.int8)
     a = rng.choice(encoded_nucleotides, size=int(length))
     return a.tobytes().decode("ascii")
+
+
+#
+# Miscellaneous auxiliary methods.
+#
+
+
+def allele_remap(alleles_from, alleles_to):
+    """
```

Comment:
I'm not 100% sure we want this method to be documented. Documenting a method declares it as something we think is useful externally and is a commitment to best-effort support indefinitely so it always has to be a considered action.

Reply:
Is it good to keep two sentences about what it does? Or is too obvious enough?

```
"""
Returns an index map from the elements in one list (alleles_from) to the elements of another list (alleles_to).

If some elements in alleles_from are not in alleles_to, then indices outside of alleles_to are used.
"""
```

Reply:
Yes, fine to keep that, but format it with `#` as comments so it doesn't get picked up by sphinx.